src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -505,6 +512,50 @@ private bool AgeCookies(string domain)
             return true;
         }
 
+        private void DomainTableCleanup()
+        {
+            var removePathList = new List<Object>();
```

Comment:
Nit: Object => object

Reply:
Why a lowercase `object` is preferred? From this [link](https://stackoverflow.com/questions/11352316/whats-the-difference-between-uppercase-object-and-lowercase-object): object is a keyword (alias) for System.Object, when compiled it will be exactly the same thing.

I guess the same reason below? (avoid boxing)

Reply:
It is part of coding style: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md see [11]